modules/bonding-multiple-network-interface-single-interface.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/upi/installing-bare-metal.adoc
+// * installing/installing_bare_metal/upi/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/upi/installing-bare-metal-network-customizations.adoc
+// * installing/installing_platform_agnostic/installing-platform-agnostic.adoc
+// * installing/installing_ibm_power/installing-ibm-power.adoc
+// * installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
+// * installing/installing_ibm_z/installing-ibm-z-lpar.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z-lpar.adoc
+// * installing/installing_ibm_power/installing-ibm-power.adoc
+// * installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc
+
+ifeval::["{context}" == "installing-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-ibm-z-kvm"]
+:ibm-z-kvm:
+endif::[]
+ifeval::["{context}" == "installing-ibm-z-lpar"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z-kvm"]
+:ibm-z-kvm:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z-lpar"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-ibm-power"]
+:ibm-power:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-power"]
+:ibm-power:
+:restricted:
+endif::[]
+
+ifndef::ibm-z-kvm[]
+:_mod-docs-content-type: CONCEPT
+[id="bonding-multiple-network-interface-single-interface_{context}"]
+== Bonding multiple network interfaces to a single interface
+
+[role="_abstract"]
+As an optional task, you can bond multiple network interfaces to a single interface by using the `bond=` option. Refer to the following examples:
+
+The syntax for configuring a bonded interface is: `bond=<name>[:<network_interfaces>][:options]`
+
+`<name>` is the bonding device name (`bond0`), `<network_interfaces>` represents a comma-separated list of physical (ethernet) interfaces (`em1,em2`),
+and _options_ is a comma-separated list of bonding options. Enter `modinfo bonding` to see available options.
+
+When you create a bonded interface using `bond=`, you must specify how the IP address is assigned and other
+information for the bonded interface.
```

Comment:
Reminder to avoid hard wraps and that specific API references and variables should be followed by a noun. 

I wonder if this would be better represented similar to the following example:

```suggestion
.Example command syntax
[source,terminal]
----
$ bond=<name>[:<network_interfaces>][:<options>]
----
where:
+
`<name>`:: Specifies the bonding device name (`bond0`) 
`<network_interfaces>`:: Specifies a comma-separated list of physical (ethernet) interfaces, such as `em1,em2`.
`<options>:: Specifies a comma-separated list of bonding options. Enter the `modinfo bonding` command to see available options.

When you create a bonded interface using the `bond=` command, you must specify how the IP address is assigned and other information for the bonded interface.
```

Reply:
Also, IDK if this should be preamble or maybe the first step?